pydis_site/apps/content/resources/code-of-conduct.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Python Discord Code of Conduct
+description: The Code of Conduct for our community.
+icon: fab fa-discord
+---
+
+# Code of Conduct
+
+We are committed to providing a friendly, safe and welcoming environment for all, regardless of level of experience, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other similar characteristic.
+
+##### Examples of behavior that contributes to creating a positive environment include:
+
+* Being kind and courteous to others
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Collaborating with other community members
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+##### Examples of unacceptable behavior by participants include:
+
+*  The use of sexualized language or imagery and sexual attention or advances
+*  The use of inappropriate images, including in a community member's avatar
+*  The use of inappropriate language, including in a community member's nickname
+*  Any spamming, flaming, baiting or other attention-stealing behavior
+*  Trolling, insulting/derogatory comments, and personal or political attacks
+*  Public or private harassment
+*  Publishing others' private information, such as a physical or electronic address, without explicit permission
+*  Discussing topics that are overly polarizing, sensitive, or incite arguments. This includes the discussion of polarizing political views, violence, suicide, and rape.
+*  Responding with “RTFM”, "just google it” or similar phrases in response to help requests
+*  Other conduct which could reasonably be considered inappropriate
+
+### Our Goal
+
+The goal of this document is to set the overall tone for our community.
+This isn’t an exhaustive list of things you can and can't do.
+Rather, take this document in the spirit in which it’s intended, and try to be your best self.
+
+We value many things beyond technical expertise, including collaboration and supporting others within our community.
+Providing a positive experience for other community members can have a much more significant impact than simply providing the correct answer.
+
+### Scope
+
+This Code of Conduct applies to all spaces managed by Python Discord.
+This includes, but is not limited to, the Discord server, our repositories on GitHub, the YouTube-channel, and meet-ups.
+In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+
+The Python Code of Conduct applies equally to all members of the community, including staff.
+
+---
+
+# Code of Conduct Policies
```

Comment:
Was there a discussion about merging the CoC pages into one? Since the content is important, we should be mindful of how a longer page may detract readers. I'm not necessarily for or against it; I just want to make sure some collective thought was given to the decision.

Reply:
I merged them initially because the page had a custom sidebar, I'll bring it up with the admin team.